docs/sources/configuration/_index.md

```diff
@@ -310,6 +310,7 @@ The queryrange_config configures the query splitting and caching in the Loki que
 # CLI flag: -querier.split-queries-by-interval
 [split_queries_by_interval: <duration> | default = 0s]
 
+# Deprecated: Split queries by day and execute in parallel.
```

Comment:
```suggestion
# Deprecated: Split queries by day and execute in parallel. Instead use `-querier.split-queries-by-interval`.
```

Reply:
Change to: "Use -querier.split-queries-by-interval instead."

Reply:
Updated the note as suggested.